sdk/lib/convert/json.dart

```diff
@@ -76,6 +76,22 @@ const JsonCodec json = JsonCodec();
 ///
 /// Shorthand for `json.encode`. Useful if a local variable shadows the global
 /// [json] constant.
+///
+/// Example:
+/// ```dart
```

Comment:
Should we include imports in code samples? In this case `import 'dart:convert';`. Not sure if that's the style or not?

Reply:
Import is not included in the examples currently with a quick glance, but for example dart:async main page has the mention to import it(https://api.dart.dev/stable/2.14.2/dart-async/dart-async-library.html).